check.py

```diff
@@ -191,26 +191,6 @@ def fix(x):
     if actual != expected:
       fail(actual, expected)
 
-print '\n[ checking binaryen-shell experimental testcases... ]\n'
```

Comment:
Yes, and I see that `s2wasm` is run on those. However, we previously also ran `binaryen-shell` on the wasts, to verify that it could parse them, which your code removed.

This isn't what the tests were intended for, I suppose, but I did notice that those wasts had a bunch of stuff not in the spec wasts, so it was useful to verify binaryen can parse them.